python/test_data/SPE1CASE1a/SPE1CASE1.DATA

```diff
@@ -422,7 +422,8 @@ WCONINJE
 
 TSTEP
 --Advance the simulater once a month for TEN years:
-31 28 31 30 31 30 31 31 30 31 30 31 /
+10*1 /
```

Comment:
Maybe add a comment why the first time steps are chosen as 10 days?

Reply:
It is 10 time steps. Each time step is 1 day.

Reply:
Will add comments also.